frontend/packages/kubevirt-plugin/src/components/vms/vm-details.tsx

```diff
@@ -106,6 +145,34 @@ export const VMDetails: React.FC<VMDetailsProps> = (props) => {
         {OSMismatchAlert}
         <HashAnchor hash="details" />
         <SectionHeading text={`${kindObj.label} Details`} />
+        {kindObj === VirtualMachineModel && isVMRequireRestart && (
+          <div>
+            <Alert
+              title="Pending Changes"
+              isInline
+              variant={AlertVariant.warning}
+              className="kubevirt-vm-details__pending-changes-class-alert"
+            >
+              <List>
+                {Object.keys(vmConfChanges).map(
+                  (key) =>
+                    vmConfChanges[key] && (
+                      <ListItem key={key}>
+                        <Button onClick={() => openModal(key)} isInline variant="link">
+                          {key}
+                        </Button>
+                      </ListItem>
+                    ),
+                )}
+              </List>
+            </Alert>
+            <BootOrderModal
```

Comment:
can we refactor BootOrderModal so it can use modalLauncher as well without a need of open flag?